engine/admin/logging/overview.md

```diff
@@ -89,6 +89,26 @@ json-file
 {% endraw %}
 ```
 
+## Configure the delivery mode of log messages from container to log driver
+
+Docker provides two modes for delivering messages from the container to the log driver since Docker 17.04:
+1. (default) direct, blocking delivery from container to driver
+2. non-blocking delivery that stores log messages in an intermediate per-container ring buffer
+
+The non-blocking mode of message delivery is useful for ensuring that applications will not block when the logging sub-system is unable to keep up with message throughput.
+
+*WARNING*: the oldest messages in memory *will be dropped* when the buffer is full, but this is often preferable to blocking the log-writing process of an application.  
```

Comment:
This should be an admonition, which means leading each line with `>`. In addition, to get the warning style, add `{:.warning}` on a line by itself (with no leading `>`) before line 101.